frontend/src/stores/courses.js

```diff
@@ -9,7 +9,12 @@ const isAvailableByTime = (courseItem) =>
     // and must not be 1 hours before starting time
     dateFns.differenceInHours(courseItem.startDate, new Date()) > 1;
 
-class CourseStore {
+const hasSufficientFund = (balance, courseItem) => courseItem.price < balance;
```

Comment:
Should it be `<=` instead of `<` as having 2.5e for course costing 2.5e is sufficient?